stix2/sources/filesystem.py

```diff
@@ -19,7 +19,7 @@
 class FileSystemStore(DataStore):
     """
     """
-    def __init__(self, stix_dir="stix_data", name="FileSystemStore"):
+    def __init__(self, name="FileSystemStore", stix_dir="stix_data"):
```

Comment:
Why does this need a `name`? If we want to uniquely identify the Source/Sink/Store in a human-readable way, I'd say the absolute path to the data directory should be sufficient.

Reply:
Why do we need a `name` for any of them?